openshift/templates/fathom-proxy/fathom-proxy-build.json

```diff
@@ -17,7 +46,10 @@
       "metadata": {
         "name": "${NAME}",
         "labels": {
-          "app": "${NAME}"
+          "app": "${APP_NAME}${SUFFIX}",
+          "app-group": "${APP_GROUP}",
+          "role": "${ROLE}",
```

Comment:
I think we could remove the role altogether from the build configurations, as it is unnecessary.